core/tests/protractor_desktop/ExplorationTranslationTab.js

```diff
@@ -96,46 +148,111 @@ describe('Exploration translation', function() {
     explorationEditorTranslationTab.expectHintsTabContentsToMatch(
       ['This is hint1.', 'This is hint2.']);
     users.logout();
+    general.checkForConsoleErrors([]);
   });
 
   it('should have a correct numerical status', function() {
-    users.createUser('user2@translationTab.com', 'user2TranslationTab');
     users.login('user2@translationTab.com');
-    workflow.createExploration();
-
-    explorationEditorMainTab.setStateName('first');
-    explorationEditorMainTab.setContent(forms.toRichText(
-      'This is first card.'));
-    explorationEditorMainTab.setInteraction('NumericInput');
-    explorationEditorMainTab.addResponse(
-      'NumericInput', forms.toRichText('This is feedback1.'),
-      'second', true, 'Equals', 6);
-    var responseEditor = explorationEditorMainTab.getResponseEditor('default');
-    responseEditor.setFeedback(forms.toRichText('This is default_outcome.'));
-    explorationEditorMainTab.addHint('This is hint1.');
-    explorationEditorMainTab.addHint('This is hint2.');
-    explorationEditorMainTab.addSolution('NumericInput', {
-      correctAnswer: 6,
-      explanation: 'This is solution.'
-    });
-    explorationEditorMainTab.moveToState('second');
-    explorationEditorMainTab.setContent(
-      forms.toRichText('This is second card.'));
-    explorationEditorMainTab.setInteraction('Continue');
-    responseEditor = explorationEditorMainTab.getResponseEditor('default');
-    responseEditor.setDestination('final card', true, null);
-    // Setup a terminating state.
-    explorationEditorMainTab.moveToState('final card');
-    explorationEditorMainTab.setInteraction('EndExploration');
-    explorationEditorMainTab.moveToState('first');
-    explorationEditorPage.saveChanges();
-
+    creatorDashboardPage.get();
+    // Test using common exploration.
+    creatorDashboardPage.editExploration('tests');
+    
     explorationEditorPage.navigateToTranslationTab();
+    explorationEditorTranslationTab.exitTutorial();
+    // To check absence of any audio translation in exploration initially.
+    explorationEditorTranslationTab.expectNumericalStatusToMatch(
+      '(0/11)');
+    explorationEditorTranslationTab.openUploadAudioModal();
+    var relativePathOfAudioToUpload = '../data/cafe.mp3';
+    var audioAbsolutePath = path.resolve(
+      __dirname, relativePathOfAudioToUpload);
+    explorationEditorTranslationTab.audioUploadInputElem.sendKeys(
+      audioAbsolutePath);
+    explorationEditorTranslationTab.saveUploadedAudio(true);
+    /** To check correct display of numerical status after
+    *   the upload of audio translation in exploration.
+    */
     explorationEditorTranslationTab.expectNumericalStatusToMatch(
-      '(0/8)');
+      '(1/11)');
+    explorationEditorTranslationTab.openUploadAudioModal();
+    relativePathOfAudioToUpload = '../data/img.png';
+    audioAbsolutePath = path.resolve(__dirname, relativePathOfAudioToUpload);
```

Comment:
done.
@DubeySandeep  However, I feel using the function uploadAudio(relativePath) and then expecting the error will be better from users' perspective. Let me know if you feel otherwise. Will make the required changes.
Right now, I have just introduced the function uploadAudio(relativePath) addressing this comment.
Thanks.